packages/web/src/theme/index.ts

```diff
@@ -13,6 +13,31 @@ export const theme = extendTheme({
         color: mode('blue.800', 'gray.50')(props),
         backgroundColor: mode('gray.50', 'gray.800')(props),
       },
+      'div.navbar': {
+        backgroundColor: mode('gray.100', 'gray.900')(props),
+      },
+      'a.navbar': {
+        backgroundColor: mode('gray.100', 'gray.900')(props),
+        px: 2,
+        py: 1,
+        rounded: 'md',
+      },
+      'a.navbar:hover': {
+        textDecoration: 'none',
+        backgroundColor: mode('gray.200', 'gray.700')(props),
+      },
+      'button.navbarProfileButton': {
+        cursor: 'pointer',
+        rounded: 'md',
+        px: 1,
+      },
+      'button.navbarProfileButton:hover': {
+        textDecoration: 'none',
+        backgroundColor: mode('gray.200', 'gray.700')(props),
+      },
+      'a.navbarMenuItem:hover': {
+        textDecoration: 'none',
+      },
```

Comment:
Instead of defining this in the global section, use variants on the components you're using. This will let chakra optimize when they are used instead of always including them.

And then instead of
```
<Box className="navbar">
```
you would have
```
<Box variant="navbar">
```

https://chakra-ui.com/docs/theming/customize-theme#customizing-single-components

Reply:
Let me know if you have any issues converting these to variants or questions on how to organize it. 👍

Reply:
done, with the execption of HStack.  To the best of my understanding HStack isn't technically a component so I was unable to get the variant approach to work for it.  Maybe, I'm doing something wrong though. For now only the `div` is left in the global settings.